### PR TITLE
hopefully fix broken changelogs

### PR DESCRIPTION
**Pull request by jozefKruszynski**

### CODE REVIEW

File: dev/playground.py [hunk 1]
```diff
@@ -20,31 +18,60 @@
 server_repo = github.get_repo(f"{ORGANIZATION}/{SERVER_REPO}")
 addon_repo = github.get_repo(f"{ORGANIZATION}/{ADDON_REPO}")
 
-frontend_release = frontend_repo.get_release("2.0.15")
+frontend_release = frontend_repo.get_latest_release()
+
+pre_bool = "True"
```
Comment (MarvinSchenkel) [inline dev/playground.py:23]: This effectively hardcodes `pre_release` to `True`, is that intentional?

**Comment by MarvinSchenkel** [comment]
Just double checking, did you meant to practically update only the dev playground? The only change for the actual release code is a data type change, which might actually work out ;).

**Comment by jozefKruszynski** [comment]
Yes, the dev/playground thing is just the local stuff, but the change of data type from bool to string was the only difference I could see between this and the add-on update action, which seems to work correctly

**Comment by MarvinSchenkel** [comment]
Then LGTM :-)

**Review by MarvinSchenkel — APPROVED** [review]

**Pull request merged**